signals/IDELAY/idelay/tests/test_identifying_delays.py

```
@@ -0,0 +1,175 @@
+    test_cases,
+)
+def test_find_K_delays(
+    name: str,
+    input_signal: np.ndarray,
+    alphas: List[float],
+    delays: List[float],
+):
+    K = len(alphas)
+    if len(delays) != K:
+        assert (
+            False
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)